dockers/docker-orchagent/supervisord.conf.j2

```diff
@@ -75,6 +75,7 @@ command=/usr/bin/orchagent.sh
 priority=4
 autostart=false
 autorestart=false
+stdout_capture_maxbytes=1MB
```

Comment:
>stdout_capture_maxbytes [](http://example.com/codeflow?start=0&length=23)

What is the reason of this change?

Reply:
This config will enable stdout capture on orchagent, then systemd will convert orchagent heartbeat message to systemd PROCESS_COMMUNICATION_STDOUT event.